masakhwe/RepairHub
# #165783321 developed user registration and authentication

#### What does this PR do
Describes the general project structure
Enables new user registration and authentication of existing users
#### Description of Task to be completed?
successful registration of new users and successful login of existing users
#### How should this be manually tested?
after cloning the repo, run python manage.py runserver
open your browser and go to (http://localhost:8000/register) and enter some dummy user data.
if successsful you should be directed to the login page with a successful flash message
login with the dummy data you registered
#### Any background context you want to provide?
takes advantage of Django 2.2 inbuilt authentication system
#### What are the relevant pivotal tracker stories?
#165783321
